extension/js/common/ui/key-import-ui.ts

```diff
@@ -136,6 +136,9 @@ export class KeyImportUi {
     await this.decryptAndEncryptAsNeeded(decrypted, encrypted, passphrase);
     await this.checkEncryptionPrvIfSelected(decrypted, encrypted);
     await this.checkSigningIfSelected(decrypted);
+    if (await KeyUtil.isWithoutIdentities(encrypted)) {
```

Comment:
I think in this situation we can just directly test for `key.identities.length === 0`, the added method does not seem necessary (since it's not a complicated or tricky or otherwise confusing thing to test for).

Thanks!